### PR TITLE
Update aRMT specs

### PR DESCRIPTION
**Pull request by mpgxvii**
- Restore deleted `aRMT-1.6.1.yml`
- Add `CNS_COVID19_FOLLOWUP_V3` (CNS), `ART_COVID19_FOLLOWUP` (ART), `MEDICATION_USE` (ART) questionnaires to schema specs

### CODE REVIEW

File: specifications/active/aRMT-1.7.0.yml [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  - type: CNS_COVID19_FOLLOWUP_V3
+    topic: questionnaire_cns_covid19_followup_v3
+    value_schema: .active.questionnaire.Questionnaire
+    questionnaire_definition_url: https://raw.githubusercontent.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/master/questionnaires/cns_covid19_followup_v3/cns_covid19_followup_v3_armt.json
+  - type: ART_COVID19_FOLLOWUP
+    topic: questionnaire_art_covid19_followup
+    value_schema: .active.questionnaire.Questionnaire
+    questionnaire_definition_url: https://raw.githubusercontent.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/master/questionnaires/art_covid19_followup/art_covid19_followup_armt.json
+  - type: MEDICATION_USE
+    topic: questionnaire_medication_use
```
Comment (yatharthranjan) [inline specifications/active/aRMT-1.7.0.yml:136]: please add `art` or `adhd` to the topic to signify the medication is for that particular cohort.

File: java-sdk/build.gradle [hunk 2]
```diff
@@ -17,7 +17,7 @@ subprojects {
     apply plugin: 'idea'
 
     // Configuration
-    version = '0.5.12-SNAPSHOT'
+    version = '0.5.11.1'
```
Comment (yatharthranjan) [inline java-sdk/build.gradle:20]: Please use SNAPSHOT version for dev branch

File: specifications/active/aRMT-1.6.1.yml [hunk 3]
```diff
@@ -0,0 +1,138 @@
+name: aRMT
+vendor: RADAR
+model: aRMT-App
+version: 1.6.1
```
Comment (yatharthranjan) [inline specifications/active/aRMT-1.6.1.yml:4]: is there a reason this was restored? was this ever registered on the server and MP?

Comment (mpgxvii) [reply specifications/active/aRMT-1.6.1.yml:4]: Yes this was registered on the server. The new topic was `cns_covid19_followup_v2`. I think this accidentally got deleted.

File: specifications/active/aRMT-1.7.0.yml (hunk 1 above)
Comment (mpgxvii) [reply specifications/active/aRMT-1.7.0.yml]: This is the ART questionnaires definitions PR https://github.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/pull/126, made the changes to the name there as well.

**Pull request merged**